README.md

```diff
@@ -0,0 +1,21 @@
+This repo is home for the SPA (Single Page Application) project templates which ship as part of ASP.NET Core.
+At the moment, those are `Angular` and `React` templates.
+
+Besides these two templates, we also want this repo to serve as a knowledge-base for all community-owned SPA project templates out there.
+We expect to maintain that list in the `SpaProjectTempltes.md` file. If you know of any such project templates, which are popular, send us a PR with added information in that file, and we'll include it, it it meets the popularity bar.
```

Comment:
```suggestion
We expect to maintain that list in the `SpaProjectTempltes.md` file. If you know of any such project templates, which are popular, send us a PR with added information in that file, and we'll include it, if it meets the popularity bar.
```

Reply:
Actually, I don't think we should have a popularity bar. I care more about having a basic quality bar: HTTPS enabled, good dev experience, publish works as expected, JS framework code kept up to date. The Awesome Blazor repo uses some tags to indicate freshness that might be useful here.

Reply:
Thanks Dan. I don't want us to be in position to validate all those with every PR submission. I'd rather let the community to share feedback about how healthy this list is.
That said, I'm ok simply listing those requirements but accepting any submissions with a quick scan of the code and the repo / package. I'll then let the community react in case we made a mistake and have included something not worthy.